controllers/part/part_ctlr.go

```diff
@@ -77,8 +79,45 @@ func All(w http.ResponseWriter, r *http.Request, params martini.Params, enc enco
 			count = ct
 		}
 	}
+	//Convert given dates to ISO8601 format.
+	if qs.Get("modified-from") != "" {
+		from = qs.Get("modified-from")
+		_, err := time.Parse(time.RFC3339, from)
+		if err != nil {
+			_, err := time.Parse(time.RFC3339, from+"-06:00")
```

Comment:
...also thoughts on defaulting to UTC if no timezone given? I'm thinking that some won't know you can add the timezone for 8601 and they'll try to do it manually. If we return UTC they don't have to think "okay central time is -6 and I'm in -5 so I need to add one to the date".

I'm not sure how much we care since:
a) they can put the timezone in the request
b) I'm assuming mostly people just care about specific days and won't get that fine grained